llvm/include/llvm/CodeGen/LivePhysRegs.h

```diff
@@ -197,16 +197,16 @@ void computeAndAddLiveIns(LivePhysRegs &LiveRegs,
 
 /// Convenience function for recomputing live-in's for a MBB. Returns true if
 /// any changes were made.
-static inline bool recomputeLiveIns(MachineBasicBlock &MBB) {
+inline bool recomputeLiveIns(MachineBasicBlock &MBB) {
```

Comment:
This needs the extern definition somewhere. Maybe just move the body out of line?

Reply:
Thanks - the `static` is there for ages, I forgot to remove it.

Reply:
Yes, I mean you need to put it back or ensure this is defined somewhere

Reply:
Fixed.